speaker/main.go

```diff
@@ -62,10 +63,10 @@ func main() {
 	}
 
 	var (
-		myNode = flag.String("node-name", "", "name of this Kubernetes node")
-		host   = flag.String("host", "", "HTTP host address")
-		port   = flag.Int("port", 80, "HTTP listening port")
-		config = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
+		myNode  = flag.String("node-name", "", "name of this Kubernetes node")
+		host    = flag.String("host", "", "HTTP host address")
+		port    = flag.Int("port", 80, "HTTP listening port")
+		mlbconf = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
```

Comment:
why the variable name change?  I'd limit the commit to what's necessary to add the feature, and optionally put any refactoring in another commit.

Reply:
This was because config variable conflicts with the imported "go.universe.tf/metallb/internal/config" library.
I'll commit this in annother PR